tests/numpy/test_np_stats.py

```diff
@@ -58,6 +58,19 @@ def test_stats_1d(nps_app_inst):
     from nums.core import application_manager
     import nums.core.settings
 
-    nums.core.settings.system_name = "serial"
+    # from mpi4py import MPI
```

Comment:
Please remove commented code. This does look useful for MPI debugging, though. Can we make a note of this and add documentation to contributor section on how to properly debug MPI?